src/visualLayer/components/BrandProfile/index.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import {
+  Button,
+  Image,
+  Tabs,
+  Typography,
+} from 'antd';
+import { useNavigate } from 'react-router-dom';
+import { NewBrandResponse } from '../../../model/types/brand';
+
+// Importing components
+import IconRenderer from '../IconRenderer';
+import BrandContactDetails from './brand-contact';
+import BrandInsights from './brand-insights';
+import BrandOverview from './brand-overview';
+
+// styles
+import './brandProfile.scss';
+
+const { Title } = Typography;
+const { TabPane } = Tabs;
+
+type BrandProfileProp = {
+  brand: NewBrandResponse,
+}
+
+const createBrandProfile = ({ brand }: BrandProfileProp): () => JSX.Element | null => {
+  return function BrandProfile(): JSX.Element {
+    const navigate = useNavigate();
+    const redirectBack = () => {
+      navigate(-1);
+    };
+    return (
+      <div className="brand-profile-container">
+        <div
+          className="brand-profile-heading"
+          style={{
+            // TODO: should come from backend
+            background: 'url(https://picsum.photos/1500/600?random=2)',
```

Comment:
Avoid hardcoding use the incoming data only. If not available you can add one more field in the type of brand named as coverImage and use that value over here.